### PR TITLE
Start with audio muted not reflected in ongoing notification

### PR DESCRIPTION
**Pull request by tmoldovan8x8**

### CODE REVIEW

File: android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java [hunk 1]
```diff
@@ -39,17 +42,24 @@
...
+    private static final String EXTRA_DATA_KEY = "extraDataKey";
+    private static final String EXTRA_DATA_BUNDLE_KEY = "extraDataBundleKey";
+    private static final String IS_AUDIO_MUTED_KEY = "isAudioMuted";
 
     private final BroadcastReceiver broadcastReceiver = new BroadcastReceiver();
 
     private boolean isAudioMuted;
 
-    static void launch(Context context) {
+    static void launch(Context context, HashMap<String, Object> extraData) {
```
Comment (saghul) [inline android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java:53]: Shouldn't we keep the previous one too, for backwards compat?

**Review by saghul — APPROVED** [review]
LGTM with a suggestion.

File: android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java (hunk 1 above)
Comment (tmoldovan8x8) [reply android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java:53]: Not sure whether there could be a situation where this requires backwards compat. Do you think there might be users that use this service on other ways that we use it?

Comment (saghul) [reply android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java:53]: Maybe if they don't use our Activity? This is part of the public API, isn't it?

**Pull request merged**